tw-NEDa-backend/src/main/java/org/neda/EarthquakeController.java

```diff
@@ -6,17 +6,24 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.bind.annotation.RestController;
 
 import java.sql.Date;
 
 @RestController
 public class EarthquakeController {
 
-    @Autowired
     private EarthquakeRepository earthquakeRepository;
 
-    @RequestMapping(value = "/works/earthquake", method = RequestMethod.GET)
+
+    @Autowired
+    public EarthquakeController(EarthquakeRepository earthquakeRepository){
+        this.earthquakeRepository = earthquakeRepository;
+    }
+
+    @RequestMapping(value = "/earthquake", method = RequestMethod.GET)
+    @ResponseBody
```

Comment:
You could change the return type for the method to Response and then construct it like here: http://stackoverflow.com/questions/16232833/how-to-respond-with-http-400-error-in-a-spring-mvc-responsebody-method-returnin